version/command.go

```diff
@@ -26,7 +26,7 @@ import (
 // ```go
 //	rootCmd.AddCommand(version.Version())
 // ```
-func Version() *cobra.Command {
+func Version(fontName string) *cobra.Command {
```

Comment:
:grimacing: I'm not sure we can change the signature of this method at this point. This package has proven quite popular and we would be breaking the upgrade path for quite a number of projects downstream.

What do you think if we offer this same thing via another method, let's say `VersionWithFont(string)` and leave the original in place?

Reply:
will make a follow up

Reply:
OK, it's merged, but still a valid comment I'd suppose @cpanato @saschagrunert  :upside_down_face:

Reply:
Yes, let's follow-up by changing it to become non-breaking.